apps/functions/dns-redirecting/index.ts

```diff
@@ -42,6 +43,13 @@ export const dnsRedirecting = functions.https.onRequest(
     } else if (hostname === 'blog.angular.io') {
       response.redirect(redirectType, `https://blog.angular.dev${request.originalUrl}`);
     } else if (hostname === 'material.angular.io') {
+      if (request.originalUrl === '/assets/versions.json') {
+        const versionsData = await fetch('https://material.angular.dev/assets/versions.json').then(
+          (r) => r.json(),
+        );
+        response.set('Content-Type', 'application/json').send(JSON.stringify(versionsData));
```

Comment:
Nit: Might be worth setting the explicit response no-cache header as well. Not sure what the default is. LGTM though.

Reply:
According to the [documentation](https://firebase.google.com/docs/hosting/manage-cache), dynamic content (all function responses) are not cached by default.

Reply:
This would be more about the browser caching it incorrectly, given no cache control header